lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -584,7 +584,11 @@ struct TFGraphLowering : public SILInstructionVisitor<TFGraphLowering, GLStatus>
   ///
   /// FIXME: Dissolve this builtin into a set of finer-grained, composable
   /// features.
-  GLStatus visitTFDataset(BuiltinInst *inst);
+  template <typename Inst>
```

Comment:
if it can simplify the patch (e.g. remove templating), pls feel free to remove the support for builtin inst, and only support graph_op.

Reply:
Let me leave the code as is for now. We can clean it up when strict deabstraction is enabled by default.